faq.adoc

```diff
@@ -1,57 +1,73 @@
 = Frequently Asked Questions
 
 
-****
-If you want to apply to receive access to our APIs, please visit the https://developer.izettle.com/register[iZettle Developer Portal].
-****
-'''
+====
+If you want to apply to receive access to our APIs, please visit https://developer.izettle.com/register[https://developer.izettle.com/register]
+====
 
-=== General questions
-**What TLS version is supported?**
+== General questions
+.*_What TLS version is supported?_*
```

Comment:
Usually we don't use italic style for regular text. So what about only using a header styling (bold) here?
The same comment for other FAQ questions.